src/globals.ts

```diff
@@ -0,0 +1,27 @@
+import chokidar from 'chokidar';
+import chalk from 'chalk';
+
+const GreenLog = chalk.hex('#00FF00');
+const OrangeLog = chalk.hex('#FFA500');
+
+export const specWatcher = (filePath: string, handler: any, handlerName: string) => {
```

Comment:
```suggestion
export const specWatcher = (filePath: string, handler: Command, handlerName: string) => {
```

Please import `Command` class type :)

Reply:
Oops @magicmatatjahu my recent commit on local machine, already fixes this. 😅
Thanks for going through that again :)